debugger/breakpoints.ml

```diff
@@ -119,25 +117,13 @@ let update_breakpoints () =
          set_breakpoints !positions;
          copy_breakpoints ())
 
-let change_version version pos =
-  Exec.protect
-    (function () ->
-       current_version := version;
-       positions := pos)
-
 (* Execute given function with no breakpoint in current checkpoint. *)
 (* --- `goto' runs faster this way (does not stop on each breakpoint). *)
 let execute_without_breakpoints f =
-  let version = !current_version
-  and pos = !positions
-  in
-    change_version 0 [];
-    try
-      f ();
-      change_version version pos
-    with
-      _ ->
-        change_version version pos
+  Misc.protect_refs [Misc.R (Debugger_config.break_on_load, false);
```

Comment:
It is not immediately clear to me why we do not also
protect `breakpoints` and `breakpoint_number`.

Reply:
(To be clear: I do not think it is necessary, but it should
not cause any harm.)